toolkits/global/packages/global-status-message/README.md

```diff
@@ -119,8 +101,22 @@ To trigger announcement of a new status message, the live region must be added t
 </div>
 ```
 
-You can achieve the initial state (without the contents) by omitting the `iconURL` and `message` properties in the data. 
+#### Status changed
+
+If the status message changes, update the `message` and `type` properties. This means the screen reader will announce the new message. 
```

Comment:
```suggestion
If you change the message after the page has loaded, a screen reader would announce the new message.
```

(It's not expected the original data for the message will change in the browser, only on the server. Changes are more likely to happen with JavaScript.)